demos/linear-wave-equation/linear_wave_equation.py.rst

```diff
@@ -67,7 +67,7 @@ output file::
 Output the initial conditions::
 
   outfile = File("out.pvd")
-  outfile << phi
+  outfile.write(phi)
```

Comment:
We use no `time=0` for the initial condition, but use `time=t` for all the other time steps. What will we see in Paraview with respect to time?